_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -169,6 +169,7 @@ release candidates.*
        transaction announcements will use more bandwidth than usual
        until everyone has upgraded to wtxid-using nodes.
 
+    {:#dont-relay-taproot}
```

Comment:
This causes the ordered list numbers to reset to 1.

Reply:
Oof, thanks for catching that!